templates/zerver/help/change-your-name.md

```diff
@@ -1,20 +1,24 @@
 # Change your name
 
-It's easy to change how your name is shown in Zulip. With full Unicode support,
-you can spell your name exactly how you'd like it to be displayed.
+By default, everyone can change their name in Zulip. You can write your name
+in any language you'd like.
 
-{!go-to-the.md!} [Your account](/#settings/your-account)
-{!settings.md!}
+Organization administrators can
+[change anyone's name](/help/change-a-users-name), and
+[prevent users from changing their names](/help/restrict-name-and-email-changes). This
+is useful when users' names are managed via LDAP or another data source.
 
-2. Change your name in the **Full name** field.
+1. From your desktop, click on the **gear**
+   (<i class="icon-vector-cog"></i>) in the upper right corner.
 
-    !!! warn ""
-        **Note:** Some organization administrators manage users' names
-        via LDAP or another data source, and have disabled the ability
-        for individual users to update their names in Zulip.  If you
-        are unable to edit your name, check with your organization
-        administrators for help.
+1. Select **Settings** from the dropdown.
+
+1. Click the button with your name in it, under **User settings**.
 
-{!save-changes.md!} account settings.
+    !!! warn ""
+        **Note:** If clicking on the button doesn't work, check that your
+        organization allows name changes. Follow the steps at
+        [prevent users from changing their names](/help/restrict-name-and-email-changes)
+        and see if the relevant box is checked.
```

Comment:
This warning may be unnecessary, since there's a little tip next to the button explaining why you can't change it, and we already introduced the possibility.

Reply:
yeah. The main reason I kept it is I think conditioned coming to this page, you probably tried to change your name and it didn't work, and googled for "zulip change your name" or something.

Reply:
I'd be in favor of merging this as is, and revisiting soon. I think there is a general problem to be solved here, which is how to write help guides where what you can do depends on organizational settings.

E.g. the "Edit or delete your messages" article is a more extreme example of this.

Reply:
Yeah, seems reasonable.  Merging.